tools/check-consistency-all-points.py

```diff
@@ -159,7 +159,13 @@ def calculate_inconsistent_points(source_points, target_points, point_ids):
                 is_data_consistent = False
                 break
 
-        fetched_points = sorted(response.json()["result"], key=lambda x: x["id"])
+        if response.json().get("result"):
+            fetched_points = sorted(response.json().get("result"), key=lambda x: x["id"])
+        else:
+            print(
+                f'level=WARN msg="Unexpected response during fetching points" uri="{uri}" response="{response.json()}"'
+            )
+            fetched_points = []
```

Comment:
Maybe we should treat this as inconsistency and proceed to following nodes.

Reply:
when do we find `results` field to be empty despite with status code 200?